x-pack/plugins/lens/public/visualization_container.scss

```diff
@@ -15,3 +15,11 @@
     position: static; // Let the progress indicator position itself against the outer parent
   }
 }
+
+.lnsExpressionRendererEmbeddedError {
```

Comment:
Following our BEM naming convention, perhaps this should be `.lnsExpressionRenderer__embeddedError`, similar to the class above (`.lnsExpressionRenderer__component`).

Presuming that to be the case, should it also be nested in the group above?

Reply:
At the moment it is not nested within a `lnsExpressionRenderer`.

Reply:
If the component using this class is not nested within `lnsExpressionRenderer`, would it be better to name the class `lnsEmbeddedError` in order to avoid confusion such as the above? Just helps further separate the two.